frontend/src/views/camp/Dashboard.vue

```diff
@@ -1,121 +1,157 @@
+            />
+            {{ item.text }}
+          </template>
+        </SelectFilter>
+        <SelectFilter
+          v-model="filter.category"
+          multiple
+          :items="categories"
+          display-field="short"
+          :label="$tc('views.camp.dashboard.category')"
+        >
+          <template #item="{ item }">
+            <CategoryChip dense :category="categories[item.value]" class="mr-1" />
+            {{ categories[item.value].name }}
+          </template>
+        </SelectFilter>
+        <SelectFilter
+          v-if="multiplePeriods"
+          v-model="filter.period"
+          :items="periods"
+          display-field="description"
+          :label="$tc('views.camp.dashboard.period')"
+        />
+        <v-chip
+          v-if="
+            filter.period ||
+            (filter.responsible && filter.responsible.length > 0) ||
+            (filter.category && filter.category.length > 0)
+          "
+          label
+          outlined
+          @click="
+            filter = {
+              period: null,
+              responsible: [],
+              category: [],
+            }
+          "
+        >
+          <v-icon left>mdi-close</v-icon>
+          {{ $tc('views.camp.dashboard.clearFilters') }}
+        </v-chip>
+      </div>
+      <table
+        v-for="(periodDays, uri) in groupedScheduleEntries"
+        :key="uri"
+        class="mx-4 mt-6 mb-3"
+        style="border-collapse: collapse"
+      >
+        <caption class="font-weight-bold text-left">
+          {{
+            periods[uri].description
+          }}
+        </caption>
+        <thead :key="uri + '_head'">
+          <tr class="d-sr-only">
+            <th :id="uri + 'th-number'" scope="col">
+              {{ $tc('views.camp.dashboard.columns.number') }}
+            </th>
+            <th :id="uri + 'th-category'" scope="col">
+              {{ $tc('views.camp.dashboard.columns.category') }}
+            </th>
+            <th :id="uri + 'th-time'" scope="col">
+              {{ $tc('views.camp.dashboard.columns.time') }}
+            </th>
+            <th :id="uri + 'th-title'" scope="col">
+              {{ $tc('views.camp.dashboard.columns.title') }}
+            </th>
+            <th :id="uri + 'th-responsible'" scope="col">
+              {{ $tc('views.camp.dashboard.columns.responsible') }}
+            </th>
+          </tr>
+        </thead>
+        <tbody
+          v-for="(dayScheduleEntries, dayUri) in periodDays"
+          :key="dayUri"
+          :aria-labelledby="dayUri + 'th'"
+        >
+          <tr>
+            <th
+              :id="dayUri + 'th'"
+              colspan="5"
+              scope="colgroup"
+              style="
+                padding-top: 0.75rem;
+                font-weight: 400;
+                color: #666;
+                font-size: 0.9rem;
+                text-align: left;
+              "
+            >
+              {{ dateLong(days[dayUri].start) }}
```

Comment:
I get a `"TypeError: Cannot read properties of undefined (reading 'start')"`
not 100% sure though if it comes from this line